swagger/Platfrom API/bizaiChats.json

```diff
@@ -0,0 +1,407 @@
+      "get": {
+        "operationId": "ChatsController_getMessages",
+        "summary": "List all messages",
+        "description": "Get all messages exchanged within a specific chat",
+        "parameters": [
+          {
+            "name": "uid",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique identifier of the chat"
+          },
+          {
+            "in": "query",
+            "name": "page",
+            "schema": {
+              "type": "integer",
+              "default": 1
+            },
+            "description": "Page number of the results."
+          },
+          {
+            "in": "query",
+            "name": "per_page",
+            "schema": {
+              "type": "integer",
+              "default": 25,
+              "maximum": 100
+            },
+            "description": "How many items to return per page."
+          },
+          {
+            "in": "query",
+            "name": "sort",
+            "schema": {
+              "type": "string",
+              "default": "updated_at:desc"
+            },
+            "description": "Sort by fields, e.g., \"created_at:asc,updated_at:desc\"."
+          },
+          {
+            "in": "query",
+            "name": "filter",
+            "schema": {
+              "type": "string"
+            },
+            "description": "Filter by a field. Example: filter[created_at][gt]=2019-01-01, filter[name]=my-name"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list of messages.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "success": {
+                      "type": "boolean"
+                    },
+                    "data": {
+                      "type": "object",
+                      "properties": {
+                        "messages": {
+                          "type": "array",
+                          "items": {
+                            "$ref": "https://vcita.github.io/developers-hub/entities/bizaiMessages.json"
+                          }
+                        }
+                      }
+                    },
+                    "page": {
+                      "type": "integer"
+                    },
+                    "per_page": {
+                      "type": "integer"
+                    },
+                    "total": {
+                      "type": "integer"
+                    },
+                    "total_pages": {
+                      "type": "integer"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "api_key": []
+          }
+        ],
+        "tags": [
```

Comment:
Tags are wrong they need to reflect a group of endpoints dealing with the same entity so tag for this should be BizAIMessage, and for the chat endpoints it should be BizAIChat
The reason behind it, is the way readme and swagger groups the endpoint in the documentation.
Please apply to all tags